cx_Utils.py

```diff
@@ -167,10 +175,10 @@ def PerformDiff(sourceDir, targetDir):
             exitCode = os.WEXITSTATUS(status)
         else:
             exitCode = status
-        raise CommandExecutionFailed(command = command, exitCode = exitCode)
+        if exitCode not in(1, 0):
```

Comment:
An exit code of 1 generally implies failure! Can you provide a bit more detail on why you want to ignore exit code 1?

Reply:
Yes, 1 was added, because diff return 1 when folders are different in my system (debian, ubuntu):
For example:

    $ diff --recursive --brief "schema" "some"
    diff --recursive --brief "schema" "some"
    Only in schema: sequence
    Only in schema: synonym
    Only in schema: table
    Only in schema: view
    $ echo $?
    echo $?
    1